### PR TITLE
Fixed "More Experiments" button at the bottom of home pages

### LINKED ISSUE #3001: Fix position of the "Try our other experiments" header
To make sure that the new featured experiment UI doesn't cannibalize installations from other experiments, we'd like to make the "Or try our other experiments" header sticky to the bottom of the page if it would be off the page.

This is blocked pending UX in #2976.

### PR DESCRIPTION
**Pull request by lmorchard**
- Add "More Experiments" button to bottom of landing pages that links to
  list of experiments, hides itself when the experiments are visible

- Create Visibility component with dive that changes classname when the
  user has scrolled down or past it

- Move HomePage container and associated components into a bundle
  directory - can now add an index.css for the More Experiments button
  and associated image

- Small tweak for path globbing to make `npm test` work on Windows
  PowerShell (should work in *nix too)

Fixes #3001

### CODE REVIEW

**Comment by lmorchard** [comment]
Work in progress, insofar as [it probably needs a better icon for the button](https://github.com/mozilla/testpilot/issues/3001#issuecomment-359597774). Could probably also use a CSS animation when it appears/disappears, but I didn't quite get that to work with opacity and display: none and could use an assist there. I was also lazy and just used a hyperlink to jump to the list of experiments - could be nice to add a scrolling transition, I guess

**Comment by lmorchard** [comment]
Oh yeah, and I need to add l10n for the "More Experiments" text!

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3289/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/a9b0e3cdea6d91f362c590b171314e8852fcf49f/

**Comment by lmorchard** [comment]
Huh. Not sure why integration tests failed, will take a look today or tomorrow

**Comment by lmorchard** [comment]
Also, [here's a more specific link to the story demoing the header](http://testpilot-storybook.dev.mozaws.net/a9b0e3cdea6d91f362c590b171314e8852fcf49f/?selectedKind=HomePage&selectedStory=Fixed%20off-screen%20subtitle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3289/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/72db535110ccbd95ed215eb09d67e5e08744301f/

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3289/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/72db535110ccbd95ed215eb09d67e5e08744301f/

**Comment by lmorchard** [comment]
Finally got the integration test to pass, dropping the [WIP]

**Comment by meandavejustice** [comment]
@lmorchard This doesn't seem to have the animation from https://github.com/mozilla/testpilot/issues/3001#issuecomment-360345885

File: frontend/src/app/containers/HomePage/index.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import '../../../styles/_utils';
+
+.landingExperiments {
```
Comment (meandavejustice) [inline frontend/src/app/containers/HomePage/index.scss:3]: Missing animation from https://github.com/mozilla/testpilot/issues/3001#issuecomment-360345885

**Review by meandavejustice — CHANGES_REQUESTED** [review]
Still missing animation from https://github.com/mozilla/testpilot/issues/3001#issuecomment-360345885 https://codepen.io/youwenliang/pen/qpzOqo

**Comment by lmorchard** [comment]
> This doesn't seem to have the animation from #3001 (comment)

<strike>Yeah, [like I said](https://github.com/mozilla/testpilot/pull/3289#issuecomment-360276816), the animation is not available from UX - needs a better icon, and I can't make it</strike>

Ugh, I see: I missed the comment with the updated asset. Ended up in spam. Will take a look at adding tomorrow

**Comment by lmorchard** [comment]
Added the button animation and a string to localize for the "More Experiments" label

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3289/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/2baade5afd9f0dee7e30ed875867dd98dce73850/

**Comment by lmorchard** [comment]
Transient failure in integration test, failed to download Firefox. Re-running.

**Comment by MozillaTestPilotStorybook** [comment]
# Storybook Deployment
Pull Request: http://testpilot-storybook.dev.mozaws.net/pull/3289/index.html
Commit: http://testpilot-storybook.dev.mozaws.net/2baade5afd9f0dee7e30ed875867dd98dce73850/

**Comment by SoftVision-PaulOiegas** [comment]
@lmorchard is there a way that we could generate the Storybook links to open the exact thing that was implemented? We always click on them and they lead us to the first page (Banner > Big Banner). We need to use the search field to find out what changed and we cannot find it every time...

**Review by meandavejustice — APPROVED** [review]

**Comment by lmorchard** [comment]
>  is there a way that we could generate the Storybook links to open the exact thing that was implemented?

Not automatically, no - there's not really a way to tell what's changed. That's why I [pasted in a link to it earlier](https://github.com/mozilla/testpilot/pull/3289#issuecomment-360281117), but of course that changes after a few storybook deployments. You could copy the query string params and paste them onto a new deployment URL though

**Pull request merged**